index.html

```diff
@@ -216,14 +233,14 @@ <h1 class="formHeader">I'm always interested in hearing about new projects, so i
 				</div>
 				<div class="email formBox">
 					<label for="formMail">
-						<input class="colorbox" type="email" id="formMail" name="userEmail" placeholder="    Email address"
-							required>
+						<input class="colorbox" type="email" id="formMail" name="userEmail"
+							placeholder="    Email address" required>
 					</label>
 				</div>
 				<div class="message">
 					<label for="formMessage">
-						<textarea id="formMessage" name="text" maxlength="500"
-							placeholder="Enter text here" required></textarea>
+						<textarea id="formMessage" name="text" maxlength="500" placeholder="Enter text here"
```

Comment:
could you please set the value of name attribute to something more clear such as name="message" or name="content" to make the data collected in the form become more expicit and clear 🎊 ✔️